utils/config_build_and_install.focal.sh

```diff
@@ -8,7 +8,18 @@ fi
 
 mc_rtc_extra_steps()
 {
-  curl https://bootstrap.pypa.io/pip/2.7/get-pip.py -o get-pip.py && sudo python2 get-pip.py && rm -f get-pip.py
-  sudo pip install matplotlib
-  export MC_LOG_UI_PYTHON_EXECUTABLE=python3
+  if [ "x$PYTHON_FORCE_PYTHON2" == xON || "x$PYTHON_BUILD_PYTHON2_AND_PYTHON3" == xON]
+    curl https://bootstrap.pypa.io/pip/2.7/get-pip.py -o get-pip.py && sudo python2 get-pip.py && rm -f get-pip.py
+    sudo pip install matplotlib
+    # pip3 gets overwritten by the get-pip.py script, but the python3-pip package remains installed. This usures that pip3 remains cleanly installed.
```

Comment:
This happened on (at least) two separate laptops of the team.
In both cases the `build_and_install` script fails when trying to install python bindings using `pip3`. Upon investigation, I noticed that running `pip3` explicitely didn't work, and after forcing the reinstallation of `python3-pip` package it started working again. Re-running the script without the above patch would cause `pip3` to disappear again.

I'm not sure what causes it, and hadn't tried running using `python -m pip` at the time.